src/app/components/Account/ContractCreatorInfo.tsx

```diff
@@ -25,7 +25,8 @@ const TxSender: FC<{ scope: SearchScope; txHash: string; alwaysTrim?: boolean }>
   }
   const query = useGetRuntimeTransactionsTxHash(scope.network, scope.layer, txHash)
   const tx = query.data?.data.transactions[0]
-  const senderAddress = tx?.sender_0_eth || tx?.sender_0
+  const senderAddress =
+    tx?.signers?.[0].address_eth ?? tx?.sender_0_eth ?? tx?.signers?.[0].address ?? tx?.sender_0
```

Comment:
Is this a necessary change? If it is then it should be fixed on the API - it is too awful to use like this

Reply:
I would assume `sender_0 === signers[0]` and add a runtime check inside useGetRuntimeTransactionsTxHash:
`if (!!signers[0] && signers[0].address !== sender_0) throw new Error('Send this URL to nexus people.')`

and then I would stick to almost always using sender_0. Or always pretend sender_0 field doesn't exist (is signers field ever missing?).

Reply:
>  it should be fixed on the API

It _has been_ fixed in the API.

The `signers` array is the new, more correct solution. It has been introduced recently, deployed this morning. It will always exist, from now on. The old `sender_0` field has now been marked as deprecated, and will eventually disappear.

We only need to look at both for a short, transitional period. In a few weeks, when we can be sure that all deployed Nexus instances have v0.5.3 or newer, we can completely remove the fallback to the old, deprecated field.

Reply:
>  The old `sender_0` field has now been marked as deprecated, and will eventually disappear.

Btw the `deprecated` flag is tunneled all the way via Orval, so we get this in the IDE:

![image](https://github.com/user-attachments/assets/b48ce55b-ee56-4fe5-b688-d7b7fce44643)

So, again, we can get rid of the deprecated fields soon-ish, simplifying this significantly.

Reply:
Ah, good

(if it will take a few months: could move `signers = signers ?? [{address_eth: sender_0_eth, address: sender}]` into useGetRuntimeTransactionsTxHash)

Reply:
Good idea. Moved all the legacy-data transformation to the API layer, with appropriate TODO comment.